src/app/services/projects.service.ts

```diff
@@ -44,6 +45,14 @@ export class ProjectsService {
       });
   }
 
+  deleteUserFromProject(proj: Project, uname: string): void {
+    this.http.delete(environment.apiUrl + `/projects/${proj.id}/users/${uname}/`)
+      .subscribe( (resp) => {
+        this.getProjectUsers(proj);
+      },error => {
+      this.notificationsService.showErrorToast('Unable to delete user');
```

Comment:
@reptillicus I want to do a follow on to this in oder to hide the delete button if the user can't be deleted. I would do separate PRs to frontend/backend.